package/yast2.spec

```diff
@@ -17,7 +17,8 @@
 
 
 Name:           yast2
-Version:        4.2.78
+Version:        4.2.79
+
```

Comment:
is it intentional new line? usually it is not here as it is related to Release later

Reply:
I probably left it by error when adapted the fixing conflict in the spec file. Removing it